benchmarks/build.gradle

```diff
@@ -34,6 +34,8 @@ apply plugin: 'com.github.johnrengelman.shadow'
 // have the shadow plugin provide the runShadow task
 apply plugin: 'application'
 
+tasks.remove(assemble)  // Not published so no need to assemble
```

Comment:
How do you live with two spaces after between the content and the comment? It 🔥 my 👀. 😛